src/types/transaction_id.ts

```diff
@@ -0,0 +1,23 @@
+const trxIdRegex = /^([a-zA-Z]|[0-9]|-|_){43}$/;
```

Comment:
It can be simplified
```suggestion
const trxIdRegex = /^([a-zA-Z0-9]|-|_){43}$/;
```

Reply:
Also, confirm this: are underscores allowed?

Reply:
`\w` is the same than `[a-zA-Z0-9]|_`
```suggestion
const trxIdRegex = /^(\w|-){43}$/;
```

Reply:
46f46fe